### PR TITLE
Change to binary classification

### PR DESCRIPTION
**Pull request by dafnevk**
Fixes issue #16

### CODE REVIEW

File: notebooks/3-Model-pipeline.ipynb [hunk 1]
```diff
@@ -49,7 +49,7 @@
     }
```
Comment (dsmits) [inline notebooks/3-Model-pipeline.ipynb:49]: <p>The answer to the exercise isn't really correct anymore. The highest values aren't necessarily on the diagonal.</p><p> </p><p>I propose a new answer:</p><p>While the majority of the false predictions are done correctly (true negatives), it is relatively bad at predicting positive labels. This could be due to the fact that the dataset is skewed.</p><p> </p><p>However, we predict on the same data that we trained on. To get truly a fair estimate of the perfomance, we need to apply it on a new data set, such as the test set.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/esciencecenter-digital-skills/SICSS-odissei-machine-learning/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (dsmits) [inline notebooks/3-Model-pipeline.ipynb:49]: <p>The confusion matrix seems to me quite similar.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/esciencecenter-digital-skills/SICSS-odissei-machine-learning/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

Comment (dsmits) [inline notebooks/3-Model-pipeline.ipynb:49]: <p>The point of overfitting is less clear on this example because the results on training and test set are quite similar. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/esciencecenter-digital-skills/SICSS-odissei-machine-learning/pull/38/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

**Comment by dafnevk** [comment]
Completely true. We can try to replicate [this notebook](https://github.com/florian-huber/weather_prediction_dataset/blob/main/notebooks/machine_learning_classification_bbq.ipynb) which uses a random forest. However, than we need to explain random forests in the theory on classification in #18  /

**Review by dsmits — APPROVED** [review]
